semconvgen/generator.go

```diff
@@ -379,6 +388,31 @@ var capitalizations = []string{
 	"OTel",
 }
 
+func capitalizations(capitalizationsPath string) ([]string, error) {
+	c := append([]string(nil), staticCapitalizations...)
+
+	if capitalizationsPath != "" {
+		// #nosec G304
```

Comment:
It is a good practice to add justification

```suggestion
		// #nosec G304 -- We expect the file path to be provided by the user.
```